crates/starknet-server/src/api/json_rpc/mod.rs

```diff
@@ -102,12 +102,32 @@ impl<T: Serialize> ToRpcResponseResult for RpcResult<T> {
                     message: err.to_string().into(),
                     data: None,
                 },
+                err @ ApiError::TypesError(_) => RpcError {
+                    // random error coe
+                    code: server::rpc_core::error::ErrorCode::ServerError(52),
```

Comment:
There is a typo `coe` -> `code`, but more importantly, we shouldn't return random error codes